internal/winapi/jobobject.go

```diff
@@ -80,9 +86,68 @@ type JOBOBJECT_BASIC_PROCESS_ID_LIST struct {
 	ProcessIdList             [1]uintptr
 }
 
+// AllPids returns all the process Ids in the job object.
+func (p *JOBOBJECT_BASIC_PROCESS_ID_LIST) AllPids() []uintptr {
+	return (*[(1 << 27) - 1]uintptr)(unsafe.Pointer(&p.ProcessIdList[0]))[:p.NumberOfProcessIdsInList:p.NumberOfProcessIdsInList]
```

Comment:
why are there two `:p.NumberOfProcessIdsInList` here?

Reply:
This was just pulled directly from another similar use case in x/sys/windows where they're getting all of the items in a buffer. AFAIK the second half can just be removed and it can be `[:p.NumberOfProcessIdsInList]`